HttpFs/HttpFs.fs

```diff
@@ -987,6 +988,10 @@ module Client =
     /// Disables cookies, which are enabled by default
     let cookiesDisabled request =
       { request with cookiesEnabled = false }
+    
+    /// Disables automatic following of redirects, which is enabled by default
+    let autoFollowRedirectsDisabled request =
```

Comment:
Use a verb first "disableAutoFollowRedirects"

Reply:
Sure I will, I was using the same name as the previous one.

Reply:
This [method](https://github.com/haf/Http.fs/blob/releases/v4.x/HttpFs/HttpFs.fs#L981) is the one I didn't wanted to break